corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html

```diff
@@ -104,18 +104,5 @@ <h4>
 {% endif %}
 
 {% if downgrade_messages and not is_same_edition %}
-  <hr />
-  <h4 class="text-center">
-    {% trans 'Note: Selecting this plan will result in the following changes to your project.' %}
-  </h4>
-  {% for message in downgrade_messages %}
-    <div class="alert alert-warning alert-downgrade">
-      {{ message.message }}
-      <ul>
-        {% for detail in message.details %}
-          <li>{{ detail }}</li>
-        {% endfor %}
-      </ul>
-    </div>
-  {% endfor %}
+  {% include 'accounting/partials/downgrade_messages.html' %}
```

Comment:
Can we use either the ["include with" or "include with only"](https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#include) syntax here? Historically, I've found it hard to trace through multiple template layers to find what defines a variable. In this case, _downgrade_messages.html_ is tightly defined to only care about `downgrade_messages`, so I think explicitly passing that variable to the template here would make it easier to reason about

Reply:
I hadn't seen that syntax before, but I think it makes sense here for sure. Thanks!